Add email for SR2022 volunteer schedules (#55)
srobo/volunteer-emails: SR2022/2022-04-17-volunteer-schedule.md

---

> to: Competition Volunteers

Reviewer: To match the previous emails

```suggestion
to: Competition signups
```

> from: Andy B-S's email directly

Reviewer: ```suggestion
from: Andy B-S
```

subject: Your Volunteer Roles at the competition.
preview-text: Here's all you need to know about what you're scheduled to do at the competition.
---

Hi,

> We've allocated your volunteer roles during the competition, however these roles aren't fixed. There's a chance we may need to make alterations in the days leading up to the competition, so be sure to check the links again closer to the date to make sure you're still scheduled for the same roles! I'll also try to email you when changes happen.

Reviewer: This makes it sound like they should be alarmed if something changes, which isn't necessarily true. What we want is for them just to keep an eye on it.

```suggestion
We've allocated your volunteer roles during the competition, however these roles aren't fixed. There's a chance we may need to make alterations in the days leading up to the competition, so be sure to check the links again closer to the date to double check what you're scheduled for! I'll also try to email you when changes happen.
```

Author: Apologies, I've already sent the mail

- [Thursday (set-up day)](REDACTED)
- [Friday (set-up day)](REDACTED)
- [Saturday](REDACTED)
- [Sunday](REDACTED)

We have [a description of all roles](REDACTED) including links to more documentation, to help out I've put the description of your roles below:

> {Short personal text explaining what they are signed up for and copy/pasting the role descriptions for their role}.

Reviewer: Question: Do you really want to send this personalized? That sounds like a lot of overhead for a tiny bit of customization

Author: Too late, already done :)

Author: I definitely think it's worth it, it's basically a chance for me to give every volunteer a tiny briefing, explain why they have their roles, and link them to the correct documentation for their specific roles

> 

Reviewer: Suggestion: We should end this email with another thanks for people giving up their time. Makes all the difference.

Author: Apologies, I've already sent the mail

> If you have any questions on any of your assigned roles in any way, now's the time to speak out! Just reply to this email and I'll try to cater for your preferences. As I said, we're pretty tight on volunteers right now, so don't expect a miracle!

Reviewer: It's probably worth explicitly mentioning in here for people to let us know whether their availability has changed, just in case.

Author: Apologies, I've already sent the mail
